### PR TITLE
Add Canada Hydrometric integration

### PR DESCRIPTION
**Pull request by michaeldavie**
**Description:**

Add documentation for Canada Hydrometric integration.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30426

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by michaeldavie** [comment]
Sorry about removing the workflow file, I couldn't seem to get the branch pushed otherwise.

**Review by frenck — CHANGES_REQUESTED** [review]
Please restore the workflow.

**Comment by michaeldavie** [comment]
Fixed it.

File: source/_integrations/canada_hydrometric.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The `canada_hydrometric` integration provides hydrometric data for Canadian water features from [Environment Canada](https://wateroffice.ec.gc.ca/index_e.html).
+
+## Location Selection
+
+The platform automatically determines the closest hydrometric station. However, in some cases you may need to override the automatic selection to use the desired station.
+
+The location to use is determined according to the following hierarchy:
+
+  - Station ID specified in platform configuration (optional)
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]:
```suggestion
- Station ID specified in platform configuration (optional)
```

File: source/_integrations/canada_hydrometric.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - Closest station to latitude/longitude specified in platform configuration (optional)
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]:
```suggestion
- Closest station to latitude/longitude specified in platform configuration (optional)
```

File: source/_integrations/canada_hydrometric.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - Closest station to latitude/longitude specified in Home Assistant core configuration
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]:
```suggestion
- Closest station to latitude/longitude specified in Home Assistant core configuration
```

File: source/_integrations/canada_hydrometric.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - Sensor
+ha_release: 0.104
+ha_iot_class: Cloud Polling
+---
+
+The `canada_hydrometric` integration provides hydrometric data for Canadian water features from [Environment Canada](https://wateroffice.ec.gc.ca/index_e.html).
+
+## Location Selection
+
+The platform automatically determines the closest hydrometric station. However, in some cases you may need to override the automatic selection to use the desired station.
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]:
```suggestion
The platform automatically determines the closest hydrometric station. However, in some cases, you may need to override the automatic selection to use the desired station.
```

File: source/_integrations/canada_hydrometric.markdown [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  type: float
+longitude:
+  description: Part of a set of coordinates to use when finding the closest station.
+  required: inclusive
+  type: float
+station:
+  description: The station code of a specific station to use. If provided, this station will be used and any latitude/longitude coordinates provided will be ignored. Station codes must be in the form of `ON/02KF005`, where `ON`is a provincial abbreviation and `02KF005` is a station code.
+  required: false
+  type: string
+scan_interval:
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]: `scan_interval` is not provided by this integration, but part of the entity component used.
Therefore, it should not be documented right here.

Please remove it.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/canada_hydrometric.markdown [hunk 6]
```diff
@@ -13,13 +13,13 @@ The `canada_hydrometric` integration provides hydrometric data for Canadian wate
...
 
-The platform automatically determines the closest hydrometric station. However, in some cases you may need to override the automatic selection to use the desired station.
+The platform automatically determines the closest hydrometric station. However, in some cases, you may need to override the automatic selection to use the desired station.
 
 The location to use is determined according to the following hierarchy:
 
-  - Station ID specified in platform configuration (optional)
-  - Closest station to latitude/longitude specified in platform configuration (optional)
-  - Closest station to latitude/longitude specified in Home Assistant core configuration
+ - Station ID specified in platform configuration (optional)
```
Comment (frenck) [inline source/_integrations/canada_hydrometric.markdown]: You did not fix these, please remove indentation from the list.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request closed without merging**